### ISSUE TITLE
AttributeError: Deserializer instance has no attribute 'demangle'

### ISSUE THREAD

**Issue by hackathi** [comex/smash-stuff#1]
When deserializing a xtal which contians a SerOp.Name, the script fails with `AttributeError: Deserializer instance has no attribute 'demangle'`

As the whole function chunk is missing and I could not find any docs about the format, I decided to open a bug.

**Comment by comex** [comment]
Yeah, I sort of abandoned this only partially finished.  Might pick it up again.  Bit of a slacker, I am...
